backend/inspirehep/records/indexer/tasks.py

```diff
@@ -52,8 +52,9 @@ def batch_index(self, records_uuids, request_timeout=None):
 
 
 def process_references_for_record(record):
-    """Tries to find differences in record references and forces to reindex
-    records which reference changed to update their citation statistics.
+    """Tries to find differences in record references
```

Comment:
Docstrings have a specific format: first line is a short description, then empty line and then a more verbose description on the purpose of the function :)